src/popup.js

```diff
@@ -0,0 +1,41 @@
+let changeColor = document.getElementById('changeColor');
```

Comment:
Whenever you can, unless you need to reassign a value use const

Reply:
Yes, you are absolutely correct. The const can be used as its element and I am not changing element I am changing only its value.

Reply:
Thank you for review